django_x509/static/django-x509/js/switcher.js

```diff
@@ -1,5 +1,21 @@
 django.jQuery(function ($) {
     'use strict';
+    var select_fields = $('.field-certificate, .field-private_key').find(".readonly");
+    // select private_key/certificate field text on click
+    select_fields.on('click', function () {
```

Comment:
Do we need the space between `function` and `( )` @R9295

Reply:
@rohithasrk , yep, the jslinter says so(using .jslintrc from django-loci)

Reply:
as suggested by @nemesisdesign

Reply:
Great. I was clarifying 👍

Reply:
@rohithasrk I will probably add some tasks during Christmas holiday to ask students to find ways to set up jslint checks of our JS files automatically in the travis-ci build, so this kind of ambiguity will be solved